test/test_mbsync.py

```diff
@@ -67,6 +68,62 @@ def test_update_library(self, track_for_mbid, album_for_mbid):
         album.load()
         self.assertEqual(album.album, 'album info')
 
+    @patch('beets.autotag.hooks.album_for_mbid')
+    @patch('beets.autotag.hooks.track_for_mbid')
+    def test_message_when_skipping(self, track_for_mbid, album_for_mbid):
+        album_for_mbid.return_value = \
+            generate_album_info('album id', ['track id'])
+        track_for_mbid.return_value = \
+            generate_track_info('singleton track id',
+                                {'title': 'singleton info'})
+
+        # Test album with no mb_albumid.
+        # The default format for an album include $albumartist so
+        # set that here, too.
+        album_invalid = Item(
+            albumartist='album info',
+            album='album info',
+            path=''
+        )
+        self.lib.add_album([album_invalid])
+
+        # default format
+        with capture_log('beets.mbsync') as logs:
+            self.run_command('mbsync')
+        e = 'mbsync: Skipping album with no mb_albumid: ' + \
+            'album info - album info'
+        self.assertEqual(e, logs[0])
```

Comment:
You're relying on the real default format, which a big dangerous:
* it may be changed in the future
* another test may override its value

I advise you to first set 'list_format_album' an 'list_format_item' in the config.